src/open_inwoner/js/admin/admin-overrides/admin-overrides.js

```diff
@@ -0,0 +1,34 @@
+const JS_CONFIRM_FIELD = document.querySelector(
+  '.field-custom_javascript_confirmed'
+)
+const JS_FILE_FIELD = document.querySelector('.field-custom_javascript')
```

Comment:
Please also hide/show file info/preview if the checkbox is checked/unchecked.
```suggestion
const JS_FILE_FIELD = document.querySelector('.field-custom_javascript')
const JS_FILE_INFO = document.querySelector('.field-custom_javascript_file_info')
```